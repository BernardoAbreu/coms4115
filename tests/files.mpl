str_file = readfile("./tests/text_input.txt");
print(str_file);

writefile("./tests/test.txt", "Bonjour Monde!");

file1 = open("./tests/text_input.txt", "r");
bonjour = readline(file1);

close(file1);
