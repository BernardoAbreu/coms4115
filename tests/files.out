Bonjour
Monde
Hello
World
Ola
Mundo
